debian/copyright

```diff
@@ -1,59 +1,781 @@
+ scripts/mysql_fix_extensions.sh
+ scripts/mysql_setpermission.sh
+ scripts/mysqlaccess.sh
+ scripts/mysqldumpslow.sh
+ scripts/mysqlhotcopy.sh
+Copyright: 2000, 2005, 2009-2011, 2017, Oracle and/or its affiliates.
+License: LGPL-2.0
+
+Files: client/completion_hash.h
+ mysys/my_port.c
+ strings/ctype-bin.c
+ strings/ctype-eucjpms.c
+ strings/ctype-uca1400.h
+ strings/ctype-uca.h
+ strings/ctype-ucs2.c
+ strings/ctype-ujis.c
+ strings/ctype-unidata.c
+ strings/ctype-utf8.c
+ strings/dtoa.c
+Copyright: 2000-2002, 2006, MySQL AB
+  2000, 2003-2004, 2007, 2013-2014, 2017, Oracle and/or its affiliates.
+  2002-2007, MySQL AB & tommy@valley.ne.jp
+  2002, MySQL AB
+  2009, 2014, SkySQL Ab.
+  2009, 2017, 2020-2021, MariaDB Corporation.
+License: LGPL-2.0
+
+Files: scripts/mysqld_multi.sh
+License: LGPL-2.0
+
+Files: include/maria.h
+ include/myisamchk.h
+ include/my_rnd.h
+ include/thr_timer.h
+ mysys/my_safehash.c
+ mysys/my_safehash.h
+ mysys/thr_timer.c
+ sql/log_slow.h
+Copyright:   2003-2007, MySQL AB
+  2006-2008, MySQL AB, 2008-2009 Sun Microsystems, Inc.
+  2006, MySQL AB
+  2009, 2017, 2019, MariaDB Corporation.
+  2012-2014, 2017, Monty Program Ab
+License: GPL-2.0+
+
+Files: client/async_example.c
+ tests/nonblock-wrappers.h
+ tests/async_queries.c
+Copyright: 2011, Kristian Nielsen and Monty Program Ab.
+License: LGPL-2.1+
+
+Files: scripts/galera_new_cluster.sh
+License: LGPL-2.1+
+
+Files: scripts/mysqld_safe.sh
+ support-files/mysql-multi.server.sh
+ support-files/mysql.server.sh
+Copyright: Abandoned 1996 TCX DataKonsult AB & Monty Program KB & Detron HB
+License: public-domain
+
+Files: dbug/dbug.c
+ dbug/dbug_long.h
+Copyright: Abandoned, 1987, Fred Fish
+License: public-domain
+
+Files: sql/socketpair.c
+Copyright: 2007, 2010, Nathan C. Myers <ncm@cantrip.org>
+License: BSD-3-clause
+
+Files: libmysqld/libmysql.c
+ sql-common/conf_to_src.c
+ sql-common/errmsg.c
+Copyright: 2000, 2014, Oracle and/or its affiliates
+  2002, 2006, MySQL AB, 2009 Sun Microsystems, Inc.
+  2009, 2020, MariaDB Corporation
+License: GPL-2.0
+ This program is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation.
+ .
+ There are special exceptions to the terms and conditions of the GPL as it
+ is applied to this software.
+ .
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+Files: strings/ctype-tis620.c
+Copyright: 1989, 1991, Samphan Raruenrom <samphan@thai.com>
+  1998, Theppitak Karoonboonyanan <thep@links.nectec.or.th>
+  1998-1999, Pruet Boonma <pruet@eng.cmu.ac.th>
+  2000, 2011, Oracle and/or its affiliates.
+  2001, Korakot Chaovavanich <korakot@iname.com> and
+  2003, Sathit Jittanupat
+  2009, 2020, MariaDB Corporation.
+License: NTP
```

Comment:
No, it's not. It is closer to "NTP No Attribution" but that isn't right either. Not even sure what this license is.